packages/generator-react-server/generators/app/index.js

```diff
@@ -14,6 +14,11 @@ module.exports = yeoman.Base.extend({
 			name: 'name',
 			message: 'What would you like to call your app?',
 			default: this.appname
+		},{
```

Comment:
@doug-wade - How did you track that down?  I don't see anything about it in travis...